xamarin/Xamarin.Forms
# Including file for testing issue 9451

### Description of Change ###

No changes, just adding the issue to the list of issues. I could not reproduce this in android/ios, though there is another issue I noticed - on Android my button was still visible, even though the parent's stacklayout's width went to 0.

### Issues Resolved ### 

- fixes #9451

### API Changes ###

 None

### Platforms Affected ### 

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Go to issue 9451 page and click the button. The stacklayout's width should go to zero.

### PR Checklist ###
